src/CypherTools.Core/DataAccess/Migrations/20200623023738_AddSourceAndRuleset.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
Hey, dotnet has migrations! That's awesome!

Reply:
Yup, though the issue i needed to solve is that it needs a runtime project.  So I had to add just a console app project that would actually do the "migrations".